Category seo text

DevertNet opened the pull request:
Pictures say more than thousand words ;)

*Important* What we do with the old "seo_text" attribute?

![bildschirmfoto 2015-10-17 um 20 45 05](https://cloud.githubusercontent.com/assets/1684349/10560638/139cb60e-7511-11e5-92a4-3f9b67b9b21c.png)

![bildschirmfoto 2015-10-17 um 20 45 24](https://cloud.githubusercontent.com/assets/1684349/10560621/d85df03a-7510-11e5-91fd-35afb80fa94d.png)

riker09 commented on app/code/community/Loewenstark/Seo/Block/Config/CategoryAttributes.php:
```diff
@@ -0,0 +1,47 @@
+        $this->_addButtonLabel = Mage::helper('loewenstark_seo')->__('Add');
+    }
+    
+    /**
```
Can you please remove the comment lines?

riker09 commented:
Interesting approach, but if I read it correctly (I don't have access to my dev box right now) on each save in the system configuration the all existing seo text attributes are deleted first. This will also delete any previously saved values in the categories.

The removal should only happen when the new list with attributes differs from the current state in the database.

For a slightly better overview you can put the prefix of the attributes in a constant of your helper.

DevertNet commented:
I fixed your code notes and added the german translation ;)

> Interesting approach, but if I read it correctly (I don't have access to my dev box right now) on each  save in the system configuration the all existing seo text attributes are deleted first. This will also delete any previously saved values in the categories.
Thats already included. It will only delete attributes which are not found in the config field (second screenshot).

riker09 commented:
Just a thought, but maybe the module should render a warning of some kind when attributes are removed or renamed in the system configuration, that all data in those fields will be lost.

There's also a slight problem with the length of the attribute code. Magento has a hardcoded max length of 30 chars for attribute codes. With the current prefix `loe_seo_text_` there are already 13 chars in use, which leaves only 17 for personalisation. The config field really should make use of some sort of validation mechanism, that doesn't allow for more than 30 characters max (prefix + user input).
